icd/api/pipeline_compiler.cpp

```diff
@@ -1050,26 +1050,79 @@ void PipelineCompiler::GetPipelineCreationInfoNext(
 
 // =====================================================================================================================
 VkResult PipelineCompiler::SetPipelineCreationFeedbackInfo(
-    const VkPipelineCreationFeedbackCreateInfoEXT* pPipelineCreationFeadbackCreateInfo,
-    const PipelineCreationFeedback*                pPipelineFeedback)
-{
-    if (pPipelineCreationFeadbackCreateInfo != nullptr)
-    {
-        if (pPipelineFeedback->feedbackValid)
-        {
-            pPipelineCreationFeadbackCreateInfo->pPipelineCreationFeedback->flags |=
-                VK_PIPELINE_CREATION_FEEDBACK_VALID_BIT_EXT;
+    const VkPipelineCreationFeedbackCreateInfoEXT*
+        pPipelineCreationFeadbackCreateInfo,
+    const VkGraphicsPipelineCreateInfo* pCreateInfo,
+    const PipelineCreationFeedback* pPipelineFeedback,
+    const PipelineCreationFeedback* pStageFeedback) {
```

Comment:
nit: these argument names should be aligned

Reply:
done